### PR TITLE
[11.x] Improves PHP 8.4 compatibility

### PR DESCRIPTION
**Pull request by crynobone**
CleanShot 2024-10-16 at 13 29 57

Notice this while adding PHP 8.4 support to `laravel/pail`

### CODE REVIEW

**Comment by flavio-schoute** [comment]
@crynobone Is it correct that we all need to update al these method properties? Otherwise I will create a script to update them in batches.

CleanShot 2024-10-16 at 11 52 29

**Comment by crynobone** [comment]
That's doesn't require any changes. Please read what's new on PHP 8.4 for further detail

File: .styleci.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 php:
   preset: laravel
   version: 8.2
+  enabled:
```
Comment (GrahamCampbell) [inline .styleci.yml:4]: I could update the Laravel preset to include this, instead, if you'd prefer.

Comment (crynobone) [reply .styleci.yml:4]: That's would be great

**Pull request merged**